typescript/src/test_util.ts

```diff
@@ -1,3 +1,9 @@
+export function setup() {
```

Comment:
> const server = (window as any).merchiBackendUri ?
         (window as any).merchiBackendUri : BACKEND_URI;

Interesting, previously I feel the only scenario the above code is need is for testcases, as now you have setup
is that mean the above code can be changed to 

> const server =  BACKEND_URI